libafl/src/mutators/mopt_mutator.rs

```diff
@@ -408,8 +418,12 @@ where
         stage_idx: i32,
     ) -> Result<MutationResult, Error> {
         let mut r = MutationResult::Skipped;
-        let swarm_now = state.mopt().swarm_now;
-        state.mopt_mut().update_pilot_operator_ctr_last(swarm_now);
+        let swarm_now = state.metadata().get::<MOpt>().unwrap().swarm_now;
```

Comment:
Instead of accessing the HashMap multiple times (`O(1)` avg case - yet still a lot of actual code), I propose to get it once as `mut`, then keep the reference thorughout this function

Reply:
yeah, I am actually trying to avoid calling get/get_mut as much as possible.
but here and in some other locations (like here https://github.com/AFLplusplus/LibAFL/blob/6bc4dbb67a17bad1ebb537a8ed7569cd9676ad01/libafl/src/stages/mopt.rs#L78)

if we get MOpt once as mut (state is mutably borrowed), later we can't borrow the state again in these calls
```
        for _i in 0..self.iterations(state, input) {
            let idx = self.schedule(state, input);
            let outcome = self
                .mutations_mut()
                .get_and_mutate(idx, state, input, stage_idx)?;
```

Reply:
ah, yes, I can still reduce calling get_mut once here in where you pointed out.